crates/turborepo-lib/src/commands/mod.rs

```diff
@@ -29,9 +27,9 @@ pub(crate) mod unlink;
 pub struct CommandBase {
     pub repo_root: AbsoluteSystemPathBuf,
     pub ui: UI,
-    user_config: OnceCell<UserConfig>,
-    repo_config: OnceCell<RepoConfig>,
-    client_config: OnceCell<ClientConfig>,
+    #[cfg(test)]
+    pub global_config_path: Option<AbsoluteSystemPathBuf>,
+    pub config: OnceCell<ConfigurationOptions>,
```

Comment:
I don't think this needs to be public
```suggestion
   config: OnceCell<ConfigurationOptions>,
```